pom.xml

```diff
@@ -154,6 +154,11 @@
             <artifactId>json</artifactId>
             <version>${json.version}</version>
         </dependency>
+        <dependency>
```

Comment:
I saw your explanation, I think using a log message might not be best way to catch a startup failure, we might need to find a less hacky approach